ballerina/csv_api.bal

```diff
@@ -16,99 +16,49 @@
 
 import ballerina/jballerina.java;
 
-# Converts CSV string to subtype of record array.
+# Converts CSV string to subtype of `record{}[]` or `anydata[][]`.
 #
-# + s - Source CSV string value
+# + csvString - Source CSV string value
 # + options - Options to be used for filtering in the projection
 # + t - Target type
 # + return - On success, value belonging to the given target type, else returns an `csv:Error` value.
-public isolated function parseStringToRecord(string s, parseToRecordOption options = {}, typedesc<record{}[]> t = <>)
-      returns t|Error = @java:Method {'class: "io.ballerina.stdlib.data.csvdata.csv.Native"} external;
+public isolated function parseString(string csvString, ParseOptions options = {}, typedesc<record{}[]|anydata[][]> t = <>)
```

Comment:
We need to add a simple example for every API that we provide. Refer https://github.com/ballerina-platform/ballerina-lang/blob/9de192a1709353c6427d30fc88c7471a800d01a8/langlib/lang.value/src/main/ballerina/value.bal#L349

Reply:
Updated